styles/main.css

```diff
@@ -17,8 +20,8 @@ nav a {
 }
 
 .byline {
-    opacity: 0.5;
     color: purple;
+    color: rgba(128,0,128,0.5);
 }

```

Comment:
This works, (no marks lost), but I was wanting you to use the rgba syntax for colours.